datalad/distribution/get.py

```diff
@@ -247,15 +256,11 @@ def _install_necessary_subdatasets(
     path: str
     reckless: bool
     """
-
-    def subds_result_filter(res):
-        return res.get('status') == 'ok' and res.get('type') == 'dataset'
-
     # figuring out what dataset to start with, --contains limits --recursive
     # to visit only subdataset on the trajectory to the target path
     subds_trail = ds.subdatasets(contains=path, recursive=True,
                                  on_failure="ignore",
-                                 result_filter=subds_result_filter)
+                                 result_filter=is_ok_dataset)
```

Comment:
When adding f634bea13 (TST+BF: Update subdatasets(contains=...) calls for previous commit, 2019-10-01), I thought a function like `is_ok_dataset` might exist, but failed to find it.  Thanks for cleaning this up.